documentation/modules/changing-precopy-intervals-ui.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+[id="changing-precopy-intervals-ui_{context}"]
+= Changing precopy intervals for warm migration
+
+The snapshots for the precopy stage of warm migration are created at one-hour intervals by default.
+
+You can change the snapshot interval by editing the `forklift-controller` deployment in the {ocp} web console.
+
+.Prerequisites
+
+* You must enable link:https://kb.vmware.com/s/article/1020128[changed block tracking (CBT)] on the VMware virtual machines and virtual disks.
```

Comment:
This is a general prerequisite in order to be able to warm migrate a VMware VM, not specific to change the interval time between precopies